### PR TITLE
Translate Ruby 2.2.1 Released (ko)

### PR DESCRIPTION
**Pull request by marocchino**

### CODE REVIEW

**Comment by marocchino** [comment]
@namsk review please.

File: ko/news/_posts/2015-03-03-ruby-2-2-1-released.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+포함됩니다.([버그 #10686](https://bugs.ruby-lang.org/issues/10686) 참조)
+
+자세한 내용은
+[변경로그](http://svn.ruby-lang.org/repos/ruby/tags/v2_2_1/ChangeLog)를
+확인하세요.
+
+## 다운로드
+
+* [http://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.1.tar.bz2](http://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.1.tar.bz2)
+  SIZE:   13326768 bytes
```
Comment (JuanitoFatas) [inline ko/news/_posts/2015-03-03-ruby-2-2-1-released.md]: need to fix identation with en post.

Comment (marocchino) [inline ko/news/_posts/2015-03-03-ruby-2-2-1-released.md]: fixed.

**Comment by namsk** [comment]
Thanks :)

**Pull request merged**